src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -725,10 +727,13 @@ def _create_pgm_input_wards(self):
         pgm_sym_loads_from_ward["p_specified"][-n_wards:] = self._get_pp_attr("ward", "pz_mw") * 1e6
         pgm_sym_loads_from_ward["q_specified"][-n_wards:] = self._get_pp_attr("ward", "qz_mvar") * 1e6
 
-        #  If input data of loads has already been filled then extend it with data of motors. If it is empty and there
-        #  is no data about loads,then assign motor data to it
+        #  If input data of loads has already been filled then extend it with data of wards. If it is empty and there
+        #  is no data about loads,then assign ward data to it
         if "sym_load" in self.pgm_input_data:
-            self.pgm_input_data["sym_load"] = np.concatenate([self.pgm_input_data["sym_load"], pgm_sym_loads_from_ward])
+            symload_dtype = self.pgm_input_data["sym_load"].dtype
+            self.pgm_input_data["sym_load"] = np.concatenate(  # pylint: disable=unexpected-keyword-arg
```

Comment:
@TonyXiang8787  pylint fails here:

````
/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/power_grid_model_io/converters/pandapower_converter.py:776:46: E1123: Unexpected keyword argument 'dtype' in function call (unexpected-keyword-arg)
````

Is it okay to disable the check? The numpy function does have that keyword.

Reply:
Yes you can do that.